pkg/registry/admissionregistration/validatingadmissionpolicybinding/authz.go

```diff
@@ -85,26 +85,31 @@ func (v *validatingAdmissionPolicyBindingStrategy) authorize(ctx context.Context
 		}
 	}
 
-	paramRef := binding.Spec.ParamRef
-
-	// require that the user can read (verb "get") the referred resource.
-	attrs := authorizer.AttributesRecord{
-		User:            user,
-		Verb:            "get",
-		ResourceRequest: true,
-		Name:            paramRef.Name,
-		Namespace:       paramRef.Namespace,
-		APIGroup:        apiGroup,
-		APIVersion:      apiVersion,
-		Resource:        resource,
-	}
+	if paramRef := binding.Spec.ParamRef; paramRef.Kind == admissionregistration.BindingParamKindGlobal && paramRef.GlobalParamRef != nil {
+		globalParamRef := paramRef.GlobalParamRef
+
+		//!TODO: can't perform static check on the namespaceParamRef
+		// should we copy this to the admission controller?
```

Comment:
Even for bindings with cluster-scoped params, don't we want an authz check at point-of-use? Is there an issue to track this?

Reply:
created

https://github.com/kubernetes/kubernetes/issues/119112

Reply:
Can we simply require "list across all namespaces" permissions for the namespaceParamRef instead?  That seem very reasonable to require given the potential impact.

Reply:
Because this is in the REST handler, it needs to be addressed when the API merges, we can't put it off.  I think 

```go
		attrs := authorizer.AttributesRecord{
			User:            user,
			Verb:            "list",
			ResourceRequest: true,
			Name:            "", // list requests don't have names
			Namespace:       "", // no namespace indicates list across all namespaces
			APIGroup:        apiGroup,
			APIVersion:      apiVersion,
			Resource:        resource,
```

is what you're looking for.

Reply:
added in #119215 and moved internal changes to that PR. We can move discussion there if there are outstanding issues